leasing/report/invoice/invoicing_review.py

```diff
@@ -423,6 +440,124 @@ def get_incorrect_management_shares_data(self, service_unit_ids, cursor):
 
         return data
 
+    def get_gaps_in_billing_periods_data(self, service_unit_ids, cursor):
+
+        query = """
+            SELECT li.identifier AS "lease_identifier",
+                l.id AS "lease_id",
+                l.start_date,
+                l.end_date,
+                r.id AS "rent_id",
+                r.start_date AS "rent_start_date",
+                r.end_date AS "rent_end_date",
+                i.id AS "invoice_id",
+                i.billing_period_start_date,
+                i.billing_period_end_date
+            FROM leasing_lease l
+            INNER JOIN leasing_leaseidentifier li
+                ON l.identifier_id = li.id
+            INNER JOIN leasing_rent r
+                ON l.id = r.lease_id
+            INNER JOIN leasing_invoice i
```

Comment:
This sounds like a potential issue using inner join here if the task is to find missing invoices. Would inner join exclude leases that don't have invoices at all?